examples/platforms/nrf52811/radio.c

```diff
@@ -87,6 +90,7 @@ static otRadioIeInfo sReceivedIeInfos[NRF_802154_RX_BUFFERS];
 static otInstance *sInstance = NULL;
 
 static otRadioFrame sAckFrame;
+static bool         sAckedWithFramePending = true;
```

Comment:
Should the flag be initialized as false, which indicates two situations?

The frame pending in the sent ACK frame is not set
No ACK was sent for the received frame, either for not requested or error.
Thought?

Reply:
I'm fine with this approach, my assumption was that if ACK was not sent e.g. due to BLE activity in multiprotocol configuration, we prefer to set this flag to 1, but I think you are right, that setting it to 0 is better.

Reply:
I also decided to keep redundant if-statement and check if the frame has ACK-requested bit set for readability in `nrf_802154_received_raw` function.